src/commands/createNewProject/openAPISteps/OpenAPICreateStep.ts

```diff
@@ -0,0 +1,64 @@
+import { OpenDialogOptions, ProgressLocation, Uri, window, workspace } from "vscode";
+import { AzureWizardExecuteStep } from "vscode-azureextensionui";
+import { ProjectLanguage } from "../../../constants";
+import { ext } from "../../../extensionVariables";
+import { localize } from "../../../localize";
+import { cpUtils } from "../../../utils/cpUtils";
+import { IFunctionWizardContext } from "../../createFunction/IFunctionWizardContext";
+
+export class OpenAPICreateStep extends AzureWizardExecuteStep<IFunctionWizardContext> {
+    public priority: number;
+
+    public async execute(wizardContext: IFunctionWizardContext): Promise<void> {
+        const uris: Uri[] = await this.askDocument();
+        const uri: Uri = uris[0];
+
+        const args: string[] = [];
+        args.push(`--input-file:${uri.fsPath} --version:3.0.6314`);
+
+        if (wizardContext.language === ProjectLanguage.TypeScript) {
+            args.push('--azure-functions-typescript');
+            args.push('--no-namespace-folders:True');
+        } else if (wizardContext.language === ProjectLanguage.CSharp) {
+            args.push('--namespace:Microsoft.Azure.Stencil');
+            args.push('--azure-functions-csharp');
+        } else if (wizardContext.language === ProjectLanguage.Java) {
+            args.push('--namespace:com.microsoft.azure.stencil');
+            args.push('--azure-functions-java');
+        } else if (wizardContext.language === ProjectLanguage.Python) {
+            args.push('--azure-functions-python');
+            args.push('--no-namespace-folders:True');
+            args.push('--no-async');
+        } else {
+            throw new Error(localize("notSupported", "Not a supported language. We currently support C#, Java, Python, and Typescript"));
+        }
+
+        args.push('--generate-metadata:false');
+        args.push(`--output-folder:${wizardContext.projectPath}`);
+
+        ext.outputChannel.show();
+        await window.withProgress({ location: ProgressLocation.Notification, title: localize('generatingFunctions', 'Generating Http trigger functions from OpenAPI...') }, async () => {
+            await cpUtils.executeCommand(ext.outputChannel, undefined, 'autorest', ...args);
```

Comment:
What's the plan for users who don't have autorest installed? And when do we expect that work to be done?

Reply:
We want users to install `autorest` rather than installing it for them - in future, we can install it after asking permission from the user.

Currently, I am going to throw with [Autorest's README](https://aka.ms/autorest ) added to the error.

@JeffHollan had similar comment above - I've added that in the next iteration.